http-server/src/test/java/org/triplea/server/error/reporting/CreateIssueStrategyTest.java

```diff
@@ -0,0 +1,106 @@
+package org.triplea.server.error.reporting;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsSame.sameInstance;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.util.function.Function;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.triplea.http.client.error.report.ErrorUploadRequest;
+import org.triplea.http.client.error.report.ErrorUploadResponse;
+import org.triplea.http.client.github.issues.GithubIssueClient;
+import org.triplea.http.client.github.issues.create.CreateIssueResponse;
+
+@ExtendWith(MockitoExtension.class)
+class CreateIssueStrategyTest {
```

Comment:
The createIssueStrategy is not new code, but looked to have been missing tests. With the further addition of rate limiting logic, it was well past time to unit test it.